src/modules/70-pipeline/strings/strings.en.yaml

```
@@ -607,6 +608,7 @@ execution:
     abortedMessage: Stage "{{stageName}}" has been aborted successfully
     pausedMessage: Stage "{{stageName}}" execution will pause once the currently running steps have completed.
     resumedMessage: Stage  "{{stageName}}" has been resumed successfully
+    userMarkFailedMessage: Stage "{{stageName}}" has been marked failed.
```

Nit:
```suggestion
    userMarkFailedMessage: Stage "{{stageName}}" has been marked failed
```
This is a phrase